sample_code/ep189/rev01/t.py

```diff
@@ -0,0 +1,22 @@
+import sys
+from typing import IO
+from typing import Optional
+
+
+def output_line(
+        s: str,
+        stream: IO[str],
+        *,
+        filename: Optional[str] = None,
+) -> None:
+    # option 1
+    if filename is not None:
+        with open(filename, 'w') as f:
+            for output_stream in (f, stream):
+                output_stream.write(f'{s}\n')
+    else:
+        stream.write(f'{s}\n')
```

Comment:
`stream` on this line was `output_stream` - I guess that this was a typo, since `output_stream` is not defined in this branch.

Reply:
yeah probably a typo